crates/db_schema/Cargo.toml

```diff
@@ -16,12 +16,14 @@ doctest = false
 [features]
 full = ["diesel", "diesel-derive-newtype", "diesel-derive-enum", "diesel_migrations", "bcrypt", "lemmy_utils",
     "activitypub_federation", "sha2", "regex", "once_cell", "serde_json", "diesel_ltree",
-    "diesel-async", "deadpool"]
+    "diesel-async", "deadpool", "ts-rs"]
 
 [dependencies]
 chrono = { workspace = true }
 serde = { workspace = true }
+serde_with = { workspace = true }
 url = { workspace = true }
+url_serde = { workspace = true }
```

Comment:
What is this crate for? Seems to be unused.

Reply:
I'll remove, it turned out to be easier to add a TS specific annotation.